tools/lib/h5tools.c

```diff
@@ -81,11 +81,20 @@ const char *volnames[] = {
  *
  */
 const char *drivernames[] = {
-    [SEC2_VFD_IDX] = "sec2",       [DIRECT_VFD_IDX] = "direct", [LOG_VFD_IDX] = "log",
-    [WINDOWS_VFD_IDX] = "windows", [STDIO_VFD_IDX] = "stdio",   [CORE_VFD_IDX] = "core",
-    [FAMILY_VFD_IDX] = "family",   [SPLIT_VFD_IDX] = "split",   [MULTI_VFD_IDX] = "multi",
-    [MPIO_VFD_IDX] = "mpio",       [ROS3_VFD_IDX] = "ros3",     [HDFS_VFD_IDX] = "hdfs",
-    [ONION_VFD_IDX] = "onion",
+    [SEC2_VFD_IDX]      = "sec2",
+    [DIRECT_VFD_IDX]    = "direct",
+    [LOG_VFD_IDX]       = "log",
+    [WINDOWS_VFD_IDX]   = "windows",
+    [STDIO_VFD_IDX]     = "stdio",
+    [CORE_VFD_IDX]      = "core",
+    [FAMILY_VFD_IDX]    = "family",
+    [SPLIT_VFD_IDX]     = "split",
+    [MULTI_VFD_IDX]     = "multi",
+    [MPIO_VFD_IDX]      = "mpio",
+    [ROS3_VFD_IDX]      = "ros3",
+    [HDFS_VFD_IDX]      = "hdfs",
+    [SUBFILING_VFD_IDX] = H5FD_SUBFILING_NAME,
```

Comment:
Should we define the other names as constants as well?

Reply:
It would probably be a good idea to keep strings in one single place and use a macro everywhere else. I don't know how many VFDs follow this pattern though. Some of them may need some tweaks.

Reply:
We don't have to make all the changes here. We can have separate PRs later to address orthogonal issues like this.